utils/download-singularity-img.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
+
```

Comment:
As previously mentioned, maybe parameter for the path. Folder with the version tag might be a good idea 👍